mkosi/config.py

```diff
@@ -4783,6 +4824,9 @@ def uki_profile_transformer(
     ) -> list[UKIProfile]:
         return [UKIProfile(profile=profile["Profile"], cmdline=profile["Cmdline"]) for profile in profiles]
 
+    def artifact_output_transformer(output: str, fieldtype: type[ArtifactOutput]) -> ArtifactOutput:
```

Comment:
Is that one necessary? I think it should in principle just work with the existing `enum_transformer`, e.g. I get
```python
>>> ArtifactOutput(14)
<ArtifactOutput.kernel|initrd|partitions: 14>
```